packages/types/src/base-api.ts

```diff
@@ -68,6 +68,11 @@ export interface IBaseFileSystemSync {
      * If path is linked, returns the actual target path.
      */
     realpathSync(path: string): string
+
+    /**
+     * Rename (move) a file or a directory
+     */
+    renameSync(filePath: string, newFilePath: string): void
```

Comment:
Path might point to a directory so parameters should be `sourcePath` / `destinationPath`.
Also make sure to change all other occurrences of `path` / `newPath` to that as well, for consistency.

Reply:
That's totally valid, updated the names 👍